Makefile

```diff
@@ -233,7 +236,7 @@ changelog: ## Generate changelog
 
 install: .gopathok install.bin install.man install.cni install.systemd  ## Install binaries to system locations
 
-install.bin:
+install.bin: bin/podman
```

Comment:
This is the problem (actually the trigger; the problem is deeper down). Any invocation of `make podman` causes a recompile, because (I'm guessing) dependencies are not set up properly. In this case, if you look at specfile:375, `BUILDTAGS` gets redefined for conmon; so the forced podman relink breaks badly.

Workaround: remove this change. Although it is absolutely correct in theory, it cannot work in the existing build environment.

Reply:
That's sad.